src/wwwroot/template-editor/adapterService.js

```diff
@@ -0,0 +1,246 @@
+(function() {
+    'use strict';
+  
+    angular
+      .module('mseditor')
+      .service('adapterService', adapterService);
+  
+      adapterService.$inject = [
+      '$http',
+      '$q'
+    ];
+  
+    function adapterService($http, $q) {
+      var service = {
+        getCampaign: getCampaign,
+        campaignSaveChanges: campaignSaveChanges,
+        campaignSaveChangesAsTemplate: campaignSaveChangesAsTemplate,
+        getCampaignRssPreview: getCampaignRssPreview,
+        deleteImages: deleteImages,
+        getImagesForCampaign: getImagesForCampaign,
+        getPreviewImage: getPreviewImage,
+        getPreviewImageWithPlay: getPreviewImageWithPlay,
+        getTiendaNubeProducts: getTiendaNubeProducts,
+        getSettings: getSettings
+      };
+  
+      return service;
+
+      function getPreviewImageWithPlay(params) {
+        /*
+          Return a preview placeholder image
+        */
```

Comment:
Is this comment necessary? in that case, maybe you could use [JSDoc](http://usejsdoc.org/)